docs/en/install-upgrade/upgrading-stack.asciidoc

```diff
@@ -1,277 +1,124 @@
+and other enhancements.
 
 [[oss-stack-upgrade]]
-=== Upgrading from a pre-5.6 installation
-
-It is possible to upgrade directly to {major-version} from a pre-5.6 installation,
-but it requires a {ref}/restart-upgrade.html[full cluster restart] and you must
-manually reindex any 2.x indices you need to carry forward to {major-version}.
-
-IMPORTANT: If you use Kibana or {xpack}, you also need to upgrade the
-internal Kibana and {xpack} indices. For information about upgrading them
-after you install Elasticsearch {version}, see
-<<upgrade-internal-indices, Upgrading internal indices>>.
-
-To manually reindex a 2.x index:
-
-. Create an index with 6.x compatible mappings.
-. Use the {ref}/docs-reindex.html[reindex API] to copy documents from the
-2.x index into the new index. You can use a script to perform any necessary
-modifications to the document data and metadata during reindexing.
-. Use the {ref}/indices-aliases.html[_aliases] API to add the name of the 2.x
-index as alias for the new index and delete the 2.x index.
-
-[[upgrade-internal-indices]]
-==== Upgrading internal indices for {major-version}
-
-The format used for the internal indices used by Kibana and {xpack} has
-changed in {major-version}. Before you can run Kibana and {xpack} in {version},
-these indices must be upgraded to the new format. If you are upgrading from a
-version prior to 5.6, you must upgrade them after after installing
-Elasticsearch {version}.
-
-To get a list of the indices that need to be upgraded, install {xpack} and use
-the {ref}/migration-api-assistance.html[`_migration/assistance` API]:
-
-[source,json]
-----------------------------------------------------------
-GET /_migration/assistance
-----------------------------------------------------------
-// CONSOLE
-
-To upgrade the `.security` index:
-
-. On a single node, add a temporary superuser account to the `file` realm.
-. Use the {ref}/migration-api-upgrade.html[`_migration/upgrade`]
-API to upgrade the security index, submitting the request with the credentials
-for the temporary superuser:
-+
---
-[source,json]
-----------------------------------------------------------
-POST /_migration/upgrade/.security
-----------------------------------------------------------
-// CONSOLE
---
-
-. Delete the temporary superuser account from the file realm.
-
-You can use your regular administration credentials to upgrade the other
-internal indices using the `_migration/upgrade` API.
-
-TIP: Once you upgrade the `.kibana` index, you can run Kibana and use the
-{xpack} Reindex Helper UI to upgrade the other indices.
+=== Upgrading from 6.6 or earlier
+
+To upgrade directly to {es} {version} from versions 6.0-6.6, you must
+{ref}/reindex-upgrade.html[manually reindex] any 5.x indices you need to
+carry forward, and perform a {ref}/restart-upgrade.html[full cluster restart].
+This includes any internal indices created in 5.x, such as the `.kibana` and
+`.security*` indices.
+
+[TIP]
```

Comment:
This TIP is being added to the Elasticsearch Reference via https://github.com/elastic/elasticsearch/pull/40019, so it can be omitted here.